src/EnergyPlus/DemandManager.cc

```diff
@@ -694,7 +660,7 @@ void GetDemandManagerInput(EnergyPlusData &state)
             DemandMgr(MgrNum).Type = DemandMgr(MgrNum).Type = ManagerType::ExtLights;
```

Comment:
Not doing the `auto &demandMgr` thing in this function?

Reply:
I probably missed that one. I intended to do them all.